scripts/tests/test_check_serial_nextest_matrix.py

```
@@ -0,0 +1,169 @@
+                "------------------------------ |"
+            ),
+            (
+                "| `cargo test`               | In-process mutex; required | "
+                "Not provided by `#[serial]`    |"
+            ),
+            (
+                "| nextest (process-per-test) | Redundant-but-harmless     | "
+                "`#[file_serial]` or test-group |"
+            ),
+            "",
+        ])
```

Ruff (0.15.20)

[warning] Consider f-string instead of string join

Replace with f-string

(FLY002)